### PR TITLE
[Intel MKL] add missing attr

### PR DESCRIPTION
**Pull request by guizili0**
add missing attr for QuantizedDepthwiseConv2D

### CODE REVIEW

**Review by penpornk — APPROVED** [review]
Thank you for the PR!

**Review by penpornk — CHANGES_REQUESTED** [review]
This actually makes changes to the public API. (It fails the [api_compatibility_test](https://source.cloud.google.com/results/invocations/febc5ab8-7e2e-4e3c-9d98-49b40fd3a3aa/log).) Please update the API golden files by running:
```
$ bazel run tensorflow/tools/api/tests:api_compatibility_test -- --update_goldens True
```
More details [here](https://source.cloud.google.com/results/invocations/febc5ab8-7e2e-4e3c-9d98-49b40fd3a3aa/log).

This will also need a review from API owners. Tagging them now.

**Comment by guizili0** [comment]
@penpornk Thanks for your comments about the API, I have update the api file based on your command.

File: tensorflow/core/ops/nn_ops.cc [hunk 1]
```diff
@@ -3266,6 +3266,7 @@ REGISTER_OP("QuantizedDepthwiseConv2DWithBiasAndRelu")
     .Attr("strides: list(int)")
     .Attr(GetPaddingAttrString())
     .Attr("dilations: list(int) = [1, 1, 1, 1]")
+    .Attr("padding_list: list(int) = []")
```
Comment (alextp) [inline tensorflow/core/ops/nn_ops.cc]: This should be named paddings, to match "dilations" above

Comment (guizili0) [reply tensorflow/core/ops/nn_ops.cc]: @alextp thanks for you comments, I have update my code based on it, please help to check, Thanks.

Comment (guizili0) [reply tensorflow/core/ops/nn_ops.cc]: @alextp Sorry to revert the change, padding_list  just make consistent with other op like:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/nn_ops.cc#L2760

**Review by guptapriya — APPROVED** [review]

**Pull request merged**